pkg/server/plugin/nodeattestor/aws/spiffeid.go

```diff
@@ -5,14 +5,15 @@ import (
 	"net/url"
 	"text/template"
 
+	"github.com/aws/aws-sdk-go/aws/ec2metadata"
 	"github.com/spiffe/spire/pkg/common/idutil"
 	"github.com/spiffe/spire/pkg/common/plugin/aws"
 )
 
 var defaultAgentPathTemplate = template.Must(template.New("agent-svid").Parse("{{ .PluginName}}/{{ .AccountID }}/{{ .Region }}/{{ .InstanceID }}"))
 
 type agentPathTemplateData struct {
-	aws.InstanceIdentityDocument
+	ec2metadata.EC2InstanceIdentityDocument
```

Comment:
I was worried at first that the field names we're embedding here (and that templates might be depending on) might have changed. But we're good; they are named the same. This does expose strictly more fields to the template, which is probably ok?

Reply:
Maybe it's better to just not embed the struct, and explicitly pass fields we want to expose.  I'll make that change.